ambry-router/src/test/java/com.github.ambry.router/NonBlockingRouterTest.java

```diff
@@ -242,6 +242,31 @@ public void testRouterBasic() throws Exception {
     assertClosed();
   }
 
+  /**
+   * Test config router.get.include.non.originating.dc.replicas to be true or false.
+   */
+  @Test
+  public void testRouterGetIncludeNonOriginatingDcReplicasConfig() throws Exception {
```

Comment:
Is this test verifying whether the effect of setting the property (different effects for true and false values)? If that is hard, I would say just avoid this test altogether. I don't see a point to this test otherwise.